### PR TITLE
(PC-15303) fix(Sonar): fix some issues

### PR DESCRIPTION
**Pull request by Lucasbeneston**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-15303

- https://sonarcloud.io/project/issues?resolved=false&rules=typescript%3AS4325&types=CODE_SMELL&id=pass-culture_pass-culture-app-native
- https://sonarcloud.io/project/issues?resolved=false&rules=typescript%3AS4624&types=CODE_SMELL&id=pass-culture_pass-culture-app-native
- https://sonarcloud.io/project/issues?resolved=false&rules=typescript%3AS3317&types=CODE_SMELL&id=pass-culture_pass-culture-app-native

## Checklist

I have:

- [x] Made sure the title of my PR follows the [convention](1) `($jira) $type($scope): $summary`.
- [x] Made sure my feature is working on the relevant real / virtual devices (native and web).
- [x] Written **unit tests** native (and web when implementation is different) for my feature.

[1]: https://github.com/pass-culture/pass-culture-app-native/blob/master/doc/standards/pr-title.md

### CODE REVIEW

**Comment by Lucasbeneston** [comment]
Il y a une Security Hotspots qui remonte (déjà présente avant la modification) mais je ne comprends pas trop comment la fix
https://sonarcloud.io/project/security_hotspots?id=pass-culture_pass-culture-app-native&branch=PC-15303&resolved=false&types=SECURITY_HOTSPOT

File: src/features/search/components/SearchResults.tsx [hunk 1]
```diff
@@ -129,7 +129,13 @@ export const SearchResults: React.FC = () => {
 
   const helmetTitle =
     numberOfResults +
-    (searchState.query.length > 0 ? ` ${t`pour`} "${searchState.query}"` : '') +
+    (searchState.query.length > 0
```
Comment (yorickeando) [inline src/features/search/components/SearchResults.tsx:132]: T'as pu vérifier le design de toutes les pages dont tu refactorisais les strings ?

**Review by yorickeando — APPROVED** [review]
Ok pour moi, si tout est en ordre sur les tests (mais ça a l'air d'être le cas)

File: src/features/search/components/SearchResults.tsx (hunk 1 above)
Comment (Lucasbeneston) [reply src/features/search/components/SearchResults.tsx:132]: Oui j'ai vérifié, il y a juste un espace manquant justement sur ce changement mais je vais l'ajouter. Merci !

**Pull request merged**